stories/toast.stories.js

```diff
@@ -26,7 +26,14 @@ storiesOf('Feedback Indicators|Toaster', module)
   ))
   .add('appearance:default', () => (
     <React.Fragment>
-      <Button onClick={() => toaster.success('Successful alert!')}>
+      <Button
+        onClick={() =>
+          toaster.success('Successful alert!', {
+            hasCloseButton: false,
+            duration: 3
```

Comment:
I would rather stick with milliseconds. Also maybe we should rename duration to `durationMs` so it will be more obvious that it should be in miliseconds

Reply:
@hzub milliseconds or seconds? What do you think?

Reply:
I left it as it was, but in my opinion, milliseconds are more common use among developers.

Reply:
certainly milliseconds